### ISSUE TITLE
[a11y documentation] Benefit Hub Page Template: Review for Accessibility Considerations

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#74090]
# CAIA A11Y Support
Internal request for support, c/o @kristinoletmuskat via  [Slack Thread](https://oddball.slack.com/archives/C04JVQGEK8F/p1705698932684429) on 1/19/2024. 

## Asignees
**CAIA A11Ys**
- Jamie 
- Sara
### Benefit Hub Page Guidance
> **1/19/2024** ... For next week, I was wondering if ya’ll could take a look this draft of [Benefit Hub Page guidance](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694550504118/9ab9423b7a907602411017fd5a5fbff952ebc8bd?sender=ua641db233dc545173f8d9816) were in the process of creating from an a11y perspective, and add comments or stickies with ideas for what a11y standards we should add? Not sure if  you want to create a ticket or just have this as a potential discussion item in ya’lls a11y sync.
#### Purpose
- The purpose of this is to document our agreed-upon standards for IA of the hub pages, and the components within them, so that  when product teams want to add stuff we know exactly how to do it.
#### General Guidelines
- General Guidelines for content inclusion in benefit hub can be found at content placement criteria. 
- Supplemental information about benefits belongs in Resources and Support.

## Definition of Done

```[tasklist]
### a11y Mural Review
- [x] Add comments or stickies to the [Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694550504118/9ab9423b7a907602411017fd5a5fbff952ebc8bd?fromVisitorModal=true&sender=u9e99e829c39911c821b39988)
- [x] Are there any a11y standards missing, or need added?
- [x] Section: Breadcrumb
- [x] Section: Intro
- [x] Contact Information
- [x] Administrative Channels
- [x] Section: Get Spoke
- [x] Section: Manage Spoke
- [x] Section: More Spoke
- [x] Section: Related Benefits
- [x] Section: Promo
- [x] Report Findings to Kristin
```

**Comment by coforma-jamie** [comment]
@kristinoletmuskat we completed our review. Please check our comments in [Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694550504118/9ab9423b7a907602411017fd5a5fbff952ebc8bd?sender=u1c2b512a71f68e7b5a5c0324) (the purple stickies).

To summarize:
- **Yes,** hide the left nav on the Hub Page
- **Yes,** call the "other" spoke "Related"
- **Add** heading levels to the other sections (not just the H1 page title)
- **Add** guidance around external links in the sidebar (i.e. "Administrative Channels")
   - if a link is opening in a new window, it needs to say "(opens in new tab)"
- **Consider** adding right sidebar headings to "On this page" on mobile
   - On mobile, Promo, Contact Info, and Admin Channels are pushed below the spokes.
   - Users might not be aware that there _is_ Contact Info, etc., below the spokes.
   - Adding those headings to "On this page" - _only on mobile_ - would raise awareness.
- **Consider** adding exact DSVA components, with links, to ease handoff to developers
- **Consider** that removing the breadcrumb makes sense, but consistency between pages is very important - if you're removing here, we should try to remove on _all_ 2-level pages
